iml-agent/src/action_plugins/ltuer.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+use std::path::PathBuf;
+use tokio_fs as fs;
+
+#[cfg(not(test))]
+static CONFIGURATION_DIR: &str = "/etc/iml";
+#[cfg(test)]
+static CONFIGURATION_DIR: &str = "/tmp/etc/iml";
+
+pub async fn create_ltuer_conf(
+    (mailbox_path, fs_name, cold_pool): (String, String, String),
+) -> Result<(), ImlAgentError> {
+    fs::create_dir_all(CONFIGURATION_DIR).await?;
+
+    let contents = format!(
+        "mailbox={}\nfs_name={}\ncold_pool={}\n",
+        mailbox_path, fs_name, cold_pool
+    );
+
+    let mut path = PathBuf::from(CONFIGURATION_DIR);
+    path.push("ltuer.conf");
+
+    fs::write(path, contents).await.map_err(|e| e.into())
+}
+
+// Warning: only one test at a time can be executing,
+// because they write to the same one file
```

Comment:
While I doubt a test is needed at all, I think it can be improved by separating formatting from I/O.

Reply:
Why do you think a test isn't needed?

Reply:
No, the question would be "why a test is needed?" :) The function being tested is trivial (string formatting and IO tested elsewhere), the test has some knowledge about implementation (e. g. order of options in config file), this feel like a tautological test. Also attempt to write a test results in much more complicated code. That's why I doubt it.

Reply:
The tests are always tautological and of course they would know order of options in config file, because that's the result of the function called.

I agree this makes the code more complex, but I think we need to explore writing smaller tests and this is first step.